app/Http/Livewire/Home/Blocks.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Livewire\Home;
+
+use App\Http\Livewire\Concerns\ManagesLatestBlocks;
+use App\ViewModels\ViewModelFactory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Collection;
+use Livewire\Component;
+
+final class Blocks extends Component
+{
+    use ManagesLatestBlocks;
+
+    public array $state = [
+        'selected' => 'blocks',
+        'type'     => 'all',
+    ];
+
+    /** @phpstan-ignore-next-line */
```

Comment:
what does this ignore? 🤔 you can give it a type with phpdoc if needed if that's the issue

Reply:
I think it was originally in LatestRecords and got moved here :sadmici: updated it